src/MoneyOut.tsx

```diff
@@ -1,33 +1,23 @@
-import { useState, useEffect } from "react";
+//import Fetch from "../helpers/Fetch";
+import useFetch from "../helpers/useFetch";
 
 const MoneyOut = () => {
-    const [data, setData] = useState([]);
+  const url = './money-out.json'; // Замініть на ваш шлях до JSON
```

Comment:
Якщо це константне значення зараз, рекомендую винести його у коремий файл, назвати його великими буквами(бо така загальна домовленість), і заімпортувати його звідти.
```suggestion
  const URL = './money-out.json'; // Замініть на ваш шлях до JSON
```